### PR TITLE
[ie/jiocinema:series] Fix extraction

### PR DESCRIPTION
**Pull request by varunchopra**
### Description of your *pull request* and other information

Currently, the extractor does not correctly extract all seasons of a given TV series. The API being used right now returns up to 10 seasons.

**Examples:**

https://www.jiocinema.com/tv-shows/mtv-splitsvilla-x5/3499820
https://www.jiocinema.com/tv-shows/mtv-roadies/3498977

You can test by running `yt-dlp` on the above shows with a filter like `--match-filter "season_number=14"` and see if the match works.

<details open><summary>Template</summary> 

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check all of the following options that apply:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Fix or improvement to an extractor (Make sure to add/update tests)
- [ ] New extractor ([Piracy websites will not be accepted](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy))
- [ ] Core bug fix/improvement
- [ ] New feature (It is strongly [recommended to open an issue first](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-new-feature-or-making-overarching-changes))

</details>

### CODE REVIEW

**Comment by bashonly** [comment]
Please add a test for a >10 season series URL

**Comment by ETapk** [comment]
https://www.jiocinema.com/tv-shows/bene-bou/1/agnidev-requests-ajanta-to-stop/3157576     URL downloaded successfully but after merging video segments The video is encrypted. not playing

**Comment by ETapk** [comment]
https://www.jiocinema.com/tv-shows/bene-bou/1/chutki-credits-polok-for-shapla-s-hardwork/3220433   At this url, yt-dlp cannot download all segments.

**Comment by bashonly** [comment]
@ETapk these problems are not relevant to this PR. Please open a new issue with proper log(s)

**Review by seproDev — APPROVED** [review]

File: yt_dlp/extractor/jiocinema.py [hunk 1]
```diff
@@ -364,20 +364,25 @@ class JioCinemaSeriesIE(JioCinemaBaseIE):
             'title': 'naagin',
         },
         'playlist_mincount': 120,
+    }, {
+        'url': 'https://www.jiocinema.com/tv-shows/mtv-splitsvilla-x5/3499820',
+        'info_dict': {
+            'id': '3499820',
+            'title': 'MTV Splitsvilla X5',
```
Comment (seproDev) [inline yt_dlp/extractor/jiocinema.py:371]:
```suggestion
            'title': 'mtv-splitsvilla-x5',
```

**Review by bashonly — APPROVED** [review]

**Pull request merged**